.github/workflows/system-install.yml

```diff
@@ -1,6 +1,7 @@
 name: System Install
 
 on:
+  pull_request:
```

Comment:
```suggestion
```

(Am i understanding correctly that this should only be running manually?)

Reply:
Yup! This is just set so I can test it during development of this PR. It will be removed prior to landing.